Makefile

```diff
@@ -31,8 +53,33 @@ $(AUTOV8_DEPOT_TOOLS):
 	mkdir -p build
 	cd build; git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
 
+ifeq ($(PLATFORM),arm64.release)
+$(AUTOV8_DIR): $(AUTOV8_DEPOT_TOOLS)
+	# patch with system-installed ninja
+	cp /usr/bin/ninja build/depot_tools/
+
+	# get ARM64 clang binaries
+	cd build; wget -nc http://releases.llvm.org/7.0.1/clang+llvm-7.0.1-aarch64-linux-gnu.tar.xz; tar xf clang+llvm-7.0.1-aarch64-linux-gnu.tar.xz
+
+	# Get and build the plugin
+	cd build; wget -nc https://chromium.googlesource.com/chromium/src/+archive/lkgr/tools/clang/plugins.tar.gz; mkdir -p plugin; cd plugin; tar xf ../plugins.tar.gz; clang++ *.cpp -c -I ../clang+llvm-7.0.1-aarch64-linux-gnu/include/ -fPIC -Wall -std=c++14 -fno-rtti -fno-omit-frame-pointer; clang -shared *.o -o libFindBadConstructs.so
+
+	cp build/plugin/libFindBadConstructs.so build/clang+llvm-7.0.1-aarch64-linux-gnu/lib/
+
+	# Build an ARM64 binary of gn
+	cd build; rm -rf gn; git clone https://gn.googlesource.com/gn; cd gn; git checkout 6ae63300be3e9865a72772e4cb6e1f8f667624c4; sed -i -e "s/-Wl,--icf=all//" build/gen.py; python build/gen.py; ninja -C out
```

Comment:
looks like you should change this line to:

```
	cd build; rm -rf gn; git clone https://gn.googlesource.com/gn; cd gn; git checkout 6ae63300be3e9865a72772e4cb6e1f8f667624c4; sed -i -e "s/-Wl,--icf=all//" build/gen.py; python build/gen.py; export PATH=$$PATH:`pwd`/../clang+llvm-7.0.1-aarch64-linux-gnu/bin ; ninja -C out
```

Reply:
This change would also not be required if clang is installed as a system dependency.